### PR TITLE
SGD8-2486: Change form action styling

### PR DESCRIPTION
**Pull request by SvenVdv**
## Description
- changed input-submit from <input> to <button>
- Added refresh icon option to standalone-links
- Added icons to form actions

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the styleguide CHANGELOG accordingly.

### CODE REVIEW

File: components/31-molecules/form-actions/_form-actions.scss [hunk 1]
```diff
@@ -19,12 +19,15 @@
     margin-left: 1.5rem;
   }
 
-  > a.back {
-    margin-left: 2.25rem;
+  > a {
+    &.refresh,
+    &.back {
+      margin-left: 2.25rem;
```
Comment (lennartvava) [inline components/31-molecules/form-actions/_form-actions.scss]:
```suggestion
      margin-left: 45px;
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -30,6 +30,9 @@ Also added the `.icon-right` class for icons that are on the right-hand side.
 **Added legend-title class**
 The legend now has a `<span class="legend-title">` around the text/title/label
 
+### Input submit
+Changed `<input>` to `<button>`
```
Comment (lennartvava) [inline CHANGELOG.md]:
```suggestion
Changed `<input>` to `<button>`. This is important to be able to use the hover effect in combination with an icon.
```

File: docs/changelog.md [hunk 3]
```diff
@@ -122,6 +125,17 @@ Becomes
 ```
 
 
+### Contact details organism
```
Comment (lennartvava) [inline docs/changelog.md]: Hoe komt dit in deze PR?

**Review by lennartvava — CHANGES_REQUESTED** [review]

File: docs/changelog.md (hunk 3 above)
Comment (lennartvava) [reply docs/changelog.md]: Is in orde. Dit is de gebuilde changelog die er bij mijn commit nog niet inzat.

Comment (SvenVdv) [reply docs/changelog.md]: I know 😛 Graag gedaan

**Pull request merged**